internal/compiler/widgets/fluent-dark/std-widgets.slint

```diff
@@ -0,0 +1,16 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+import {
+    StyleMetrics, ScrollView, Button, StandardButton, TextEdit,
+    AboutSlint, AboutSlint as AboutSixtyFPS, CheckBox, SpinBox, Slider, GroupBox, TabWidgetImpl,
+    TabImpl, TabBarImpl, TabWidget, LineEdit, ListView, StandardListView, ComboBox,
+    VerticalBox, HorizontalBox, GridBox
+} from "../fluent-base/std-widgets.slint";
+
+export {
+    StyleMetrics, ScrollView, Button, StandardButton, TextEdit,
+    AboutSlint, AboutSlint as AboutSixtyFPS, CheckBox, SpinBox, Slider, GroupBox, TabWidgetImpl,
+    TabImpl, TabBarImpl, TabWidget, LineEdit, ListView, StandardListView, ComboBox,
+    VerticalBox, HorizontalBox, GridBox
+}
```

Comment:
It would be nice to be able to write just `export * from "../fluent-base/std-widgets.slint";`. I can try to implement that in the compiler.

Reply:
That would be nice, but is not required for this